components/match2/wikis/counterstrike/match_summary.lua

```diff
@@ -219,55 +218,24 @@ function CustomMatchSummary._createMap(game)
 		table.insert(team2Scores, {style = side2 and ('brkts-cs-score-color-'.. side2) or nil, score = score2})
 	end
 
-	-- Map Info
-	local mapInfo = mw.html.create('div')
-	mapInfo	:addClass('brkts-popup-spaced')
-			:wikitext(CustomMatchSummary._createMapLink(game.map, game.game))
-			:css('text-align', 'center')
-			:css('padding','5px 2px')
-			:css('flex-grow','1')
-
-	if game.resultType == 'np' then
-		mapInfo:addClass('brkts-popup-spaced-map-skip')
-	elseif game.resultType == 'draw' then
-		mapInfo:wikitext('<i>(Draw)</i>')
-	end
-
-	-- Build the HTML
-	row:addElement(MatchSummaryWidgets.GameWinLossIndicator{winner = game.winner, opponentIndex = 1})
-	row:addElement(MatchSummaryWidgets.DetailedScore{score = score(1), partialScores = team1Scores, flipped = false})
-
-	row:addElement(mapInfo)
-
-	row:addElement(MatchSummaryWidgets.DetailedScore{score = score(2), partialScores = team2Scores, flipped = true})
-	row:addElement(MatchSummaryWidgets.GameWinLossIndicator{winner = game.winner, opponentIndex = 2})
-
-	-- Add Comment
-	if not Logic.isEmpty(game.comment) then
-		row:addElement(MatchSummaryWidgets.Break{})
-		local comment = mw.html.create('div')
-		comment :wikitext(game.comment)
-				:css('margin', 'auto')
-		row:addElement(comment)
-	end
-
-	row:addClass('brkts-popup-body-game'):css('font-size', '85%'):css('overflow', 'hidden')
-
-	return row:create()
-end
-
----@param map string?
----@param game string?
----@return string
-function CustomMatchSummary._createMapLink(map, game)
-	if Logic.isNotEmpty(map) then
-		if Logic.isNotEmpty(game) then
-			return '[[' .. map .. '/' .. game .. '|' .. map .. ']]'
-		else
-			return '[[' .. map .. '|' .. map .. ']]'
-		end
-	end
-	return ''
+	local mapInfo = {
+		mapDisplayName = game.map,
+		map = game.game and (game.map .. '/' .. game.game) or game.map,
```

Comment:
potentially errors if `game.map` is nil
```suggestion
		map = game.map and game.game and (game.map .. '/' .. game.game) or game.map,
```
if any of them could be empty string we also should nilIfEmpty them

Reply:
game.map cannot be nil, there's a check for it above